README.md

```diff
@@ -1,3 +1,19 @@
-# Walkthrough Template
+# Solution Pattern Template
 
-A template that can be used to bootstrap new walkthroughs. Walkthrough content goes under `/walkthroughs`.
+A template that can be used to bootstrap new Solution Patterns. Content goes under the `/walkthroughs` folder.
+
+## Adding Managed Services
+If you would like to include managed services with your Solution Pattern, you can add them to the walkthrough.json file under `Dependencies`.<br>
```

Comment:
Should we clarify what this means? IIRC it used to mean a Fuse Online instance would be provisioned. Will that still be the case in newer releases?

Reply:
Sure, additional info would probably be helpful. @pb82 Can you provide a quick sentence or two to explain what this means for OS4?

Reply:
Yep, would be great to have a bit more clarity on valid values for it and what they do.

Reply:
Maybe something like: `Those services will be provisioned on-demand when the user starts the solution pattern. The user will be given access to either their personal instance of the service or their own login in a multi tenant service (e.g. 3scale).`

Reply:
@pb82 that would be very helpful, along with a list of valid values 👍